va/va.h

```diff
@@ -4208,11 +4210,17 @@ typedef enum {
 } VADecodeErrorType;
 
 /**
- * Client calls vaQuerySurfaceError with VA_STATUS_ERROR_DECODING_ERROR, server side returns
+ * Client calls vaQuerySurfaceError() with the VAStatus obtained from vaSyncSurface(), server side returns
  * an array of structure VASurfaceDecodeMBErrors, and the array is terminated by setting status=-1
 */
 typedef struct _VASurfaceDecodeMBErrors {
-    int32_t status; /* 1 if hardware has returned detailed info below, -1 means this record is invalid */
+    /**
+     * Status reported by server side:
+     * 1 if hardware has returned detailed info below;
+     * -1 means this record is invalid;
+     * VA_STATUS_ERROR_RESET means reset happened in the server side.
```

Comment:
why appear here? status also could be VA_STATUS_ERROR_RESET?

Reply:
Need to clear this comment, status is output value that set by driver. It should indicate to VA_STATUS_ERROR_DECODING_ERROR, VA_STATUS_ERROR_RESET, VA_STATUS_SUCCESS or -1